src/components/MissionHubButton.stories.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import MissionHubButton from './MissionHubButton'
+
+export default {
+  title: 'Components/MissionHubButton',
+  component: MissionHubButton,
+  parameters: {
+    backgrounds: {
+      default: 'black',
+      values: [
+        { name: 'grey', value: '#F2F2F2' },
+        { name: 'black', value: '#000000' },
+      ],
+    },
+  },
+}
+
+// eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
If you want, you can make an exception for this rule for `.stories.` files so we don't have to do this every time.. There's an example in .eslintrc.js for Jest files.